geometry_msgs/Point32 start
geometry_msgs/Point32 goal
navigation/PolyArray obstacle_list
---
navigation/PointArray path
bool ack